shell/common/platform_util_mac.mm

```diff
@@ -117,6 +118,17 @@ void OpenExternal(const GURL& url,
                  });
 }
 
+gfx::NativeView GetViewForWindow(gfx::NativeWindow native_window) {
```

Comment:
I recommend we note that these are added to correct debug builds, otherwise someone might delete them in the future if they think they're dead code given they have no callsites in Electron.

Reply:
I would also prefer this to have [NOTREACHED](https://source.chromium.org/chromium/chromium/src/+/main:base/notreached.h) instead to indicate these are noop implementations to avoid accidental usage.